src/librustc_mir/monomorphize/partitioning.rs

```diff
@@ -889,3 +895,146 @@ fn debug_dump<'a, 'b, 'tcx, I>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
         }
     }
 }
+
+fn collect_and_partition_mono_items<'a, 'tcx>(
+    tcx: TyCtxt<'a, 'tcx, 'tcx>,
+    cnum: CrateNum,
+) -> (Arc<DefIdSet>, Arc<Vec<Arc<CodegenUnit<'tcx>>>>)
+{
+assert_eq!(cnum, LOCAL_CRATE);
```

Comment:
wrongly indented assertion

Reply:
Oops

Reply:
this is still/again there

```suggestion
    assert_eq!(cnum, LOCAL_CRATE);
```

Reply:
Oops